music_album.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'item'
+
+class Album < Item
+  attr_accessor :on_spotify
+
+  def initialize(album_info, on_spotify)
+    super(*album_info.to_a)
+    @on_spotify = on_spotify
+  end
+
+  def can_be_archived?
+    super || @on_spotify
+  end
```

Comment:
Kindly change this method to:
```ruby
def can_be_archived?
    super && @on_spotify
  end
```
Because the requirements say the following:
- can_be_archived?() in the MusicAlbum class
  - should override the method from the parent class.
  - **should return true if parent's method returns true AND if on_spotify equals true.
otherwise, it should return false.**
Pay special attention to the line that is in **Bold**

Reply:
@enoqueJonas Thanks for your review. The mentioned changes will be implemented on the future commits on the same branch. :slightly_smiling_face: